alpenhorn/client.py

```diff
@@ -425,47 +425,72 @@ def verify(node_name, md5, fixdb, acq):
 @cli.command()
 @click.argument('node_name', metavar='NODE')
 @click.option('--days', '-d', help='Clean files older than <days>.', type=int, default=None)
+@click.option('--cancel', help='Cancel files marked for cleaning', is_flag=True)
 @click.option('--force', '-f', help='Force cleaning on an archive node.', is_flag=True)
 @click.option('--now', '-n', help='Force immediate removal.', is_flag=True)
 @click.option('--target', metavar='TARGET_GROUP', default=None, type=str,
               help='Only clean files already available in this group.')
 @click.option('--acq', metavar='ACQ', default=None, type=str,
               help='Limit removal to acquisition.')
-def clean(node_name, days, force, now, target, acq):
+def clean(node_name, days, cancel, force, now, target, acq):
     """Clean up NODE by marking older files as potentially removable.
 
+    Files will never be removed until they are available on at least two
```

Comment:
In this documentation you might want to have a longer comment about the effects of the `--cancel` option. Given it completely inverts the operation that is to be done, we might want to discuss a little more.

Reply:
I was going to and then forgot to do it before committing. I'll add now.